src/components/layout/PageContainer.tsx

```diff
@@ -1,22 +1,26 @@
-import { Typography } from '@mui/material';
-import { Box, Container, Stack } from '@mui/system';
+import { Container, ContainerProps, Stack } from '@mui/system';
 import { ReactNode } from 'react';
+import PageTitle from '@/components/layout/PageTitle';
 import { useIsMobile } from '@/hooks/useIsMobile';
 
 const PageContainer = ({
   children,
   title,
+  overlineText,
   actions,
+  maxWidth = 'lg',
 }: {
   children: ReactNode;
   title: ReactNode;
+  overlineText?: string;
   actions?: ReactNode;
+  maxWidth?: ContainerProps['maxWidth'];
 }) => {
   const isTiny = useIsMobile('sm');
   return (
     <Container
       component='main'
-      maxWidth='lg'
+      maxWidth={maxWidth}
       sx={{ px: { xs: 1, sm: 3, lg: 4 }, pt: 4, pb: 6 }}
```

Comment:
Reviewed at design standup - reduce padding when there is overline text so the main title is on the same heigh as other pages (client search, projects)
```suggestion
      sx={{ px: { xs: 1, sm: 3, lg: 4 }, pt: overlineText ? 1 : 4, pb: 6 }}
```